packages/components/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@emqx/shared-ui-components",
+  "version": "0.0.1",
+  "homepage": "https://emqx.io",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/emqx/shared-ui"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "main": "./dist/index.umd.cjs",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.umd.cjs"
+    }
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "preview": "vite preview",
+    "version:patch": "npm version patch",
+    "version:minor": "npm version minor",
+    "version:major": "npm version major",
+    "release": "npm publish"
+  },
+  "dependencies": {
+    "@emqx/shared-ui-constants": "link:../constants"
+  },
+  "devDependencies": {
+    "@emqx/shared-ui-utils": "link:../utils",
+    "@vitejs/plugin-vue": "^5.0.0",
+    "@vue/eslint-config-prettier": "^9.0.0",
+    "@vue/eslint-config-typescript": "^13.0.0",
+    "element-plus": "^2.3.0",
+    "rollup-plugin-visualizer": "^5.12.0",
+    "vue": "^3.2.0",
+    "vue-tsc": "^2.1.10"
```

Comment:
Why not use the latest package?

Reply:
Since dashboard is still using version 3.2 of vue.. it's ok that the vue used by cloud satisfies this version number.